lib/matplotlib/tests/test_axes.py

```diff
@@ -4166,6 +4166,18 @@ def test_specgram_angle():
                                 scale="dB")
 
 
+def test_specgram_fs_none():
+    """Test axes.specgram when Fs is None, should not throw error."""
+
+    try:
+        spec, freqs, t, im = plt.specgram(np.ones(300), Fs=None)
+    except Exception as e:
+        raise pytest.fail("DID RAISE {0}".format(e))
```

Comment:
You don't need the `try ... except` pytest will fail if there is an Execption; and the message does not contain more information than the exception, which will be printed anyway.

Reply:
Hmm alright, so I'll just run ```plt.specgram``` without the ```try...except``` if pytest handles it.